src/components/ComponentPages/Home/TopicsList/index.tsx

```diff
@@ -18,9 +18,17 @@ const { Search } = Input;
 
 const infoContent = (
   <>
-    <div>
-      <Title level={5}>Lorem Ipsum</Title>
-      <p>Duis aute irure dolor in reprehderit.</p>
+    <div className={styles.namespacesPopover}>
+      <Title level={5}>Namespace</Title>
+      <p>
+        Namespaces are a set of topics created for specific organizations and
```

Comment:
Aren't we getting this from the database?

Reply:
Not yet, if it's required then I will do this in coming sprint.